### ISSUE TITLE
Allow debugging AIR target.

### ISSUE THREAD

**Issue by EricBishton** [HaxeFoundation/intellij-haxe#849]
Received this email:
> From: Patrick <vectorpark@gmail.com>
> Subject: Debugging AIR apps
>
> Message Body:
> Hi, I've just gotten OpenFL/Haxe compiling with Intellij, and I'm  
> wondering about debugging for AIR. Your Debugging page suggests the  
> plugin supports it, but when I try to debug an AIR app I get:
>
> "Error running 'Test': You can debug only flash or HXCPP targets!"
>
> Am I missing something? If not, is it something you foresee adding soon?  
> Unfortunately, my app requires the AIR api so I can't get away with  
> debugging as flash and then deploying as AIR.
>
> Thanks for your time!
>

I think that we simply need to update the test that puts out that message and stops debugging.  I use the Adobe AIR debugger when testing flash targets, so this configurattion should work.

**Comment by EricBishton** [comment]
Actually, it looks like Lime has added AIR support back in as of version 5.4.0:
```
  $ haxelib run lime
  ...
   2017-08-25 22:20:42 5.4.0 : Added AIR target, GLES3 APIs, HXP improvements
  ...
```

**Comment by Vectorpark** [comment]
Hey Eric, the issue I was having as of 7/7/18 wasn’t with AIR support per se, but with the debugger not launching for an AIR project. I have not restested since, but the issue was as follows:

When I run or test a sample app as Flash it opens in the Flash Player Debugger, as expected.

If I target AIR and Run, it opens ADL, which is also what I would expect. 

If I target AIR and Debug, it throws the error: “You can debug only flash or HXCPP targets!”

**Comment by EricBishton** [comment]
Right.  The PR addresses that by checking if the OpenFL target was AIR and allowing the debug attempt to go forward; it treats AIR as a flash target, but doesn't do anything else special.  If you have installed ADL already and point to that as your "Flex SDK for flash applications debugging," then it should work.

Since the change has been approved, I'm going to merge it and you can test the "nightly" build when you get a chance.

**Comment by EricBishton** [comment]
Fixed by #897.

And, BTW, the reason that this is tied to OpenFL is that it's the only system that outputs AIR targets right now.  The Haxe compiler (currently at 4.0.preview.5) doesn't create AIR targets.
